### DISCUSSION TITLE
Issue with Windows Installation

### DISCUSSION THREAD

**Discussion by garcs2** [accert-dev/ACCERT#40] [General]
Hello, 

I am attempting to install ACCERT on my local windows machine and am running into issues with the install. I've previously installed this on a separate ubuntu machine and it worked fine but am having issues on running `database_install.py`. I've followed the instructions for the windows install but when attempting to run the script I get the following output repeated many times over:

`The error 'Unread result found' occurred at command:`

I'm unsure where I would've gone wrong, although I do understand that this being a windows install things can act strangely. My question is if there is any known issue with the current accertdb.sql file since that seems where the issue is coming from based on tracing back the error. If there's not then that sounds good, and I would appreciate some help in overcoming this install issue.

**Comment by JiaZhou-PU** [comment]
Hi Sam 
could you refer to the new document at this page https://accert.readthedocs.io/en/latest/user/install.html#id1 
In installation database. Use cmd and MySQL to try? And paste some screen shot? And Accert has some major changes recently, could you also pull the latest version as well? 
Jia

**Reply by garcs2** [reply]
Hello Jia! 

Thank you for the timely response! After following that documentation I believe I got it working, here's a screenshot after running `source accertdb.sql` and `show databases;`:
![Screenshot 2024-10-01 113041](https://github.com/user-attachments/assets/e48da7b7-4c66-452d-a1b4-0a63cb5d9cba)

I edited this post as I resolved earlier issues by forgetting to paste sonvalidxml.exe from workbench to accert bin folder. Now I get this issue when running the pytest:

=========================================================================================================================== short test summary info =========================================================================================================================== 
FAILED test_integration.py::test_integration_with_fusion_son - IndexError: list index out of range
FAILED test_integration.py::test_integration_with_pwr_son - IndexError: list index out of range
FAILED test_integration.py::test_integration_with_abr_son - IndexError: list index out of range
FAILED test_integration.py::test_integration_with_heatpipe_son - IndexError: list index out of range
======================================================================================================================== 4 failed, 29 passed in 11.83s ========================================================================================================================

However when I run the LFR example I now get it to work!

**Reply by JiaZhou-PU** [reply]
Hi Sam,
If just the integration tests, it might be some version number of the dependents, like prettytable or numpy version. I guess that the inputs should be working fine for you, if you need further debugging, you can go to the test folder and run `pytest -v test_integration.py` and paste the results, I can help you check whether it is still a database issue.
Thanks!
Jia
